src/style/app.module.css

```diff
@@ -2037,7 +2095,19 @@ input[type='radio']:checked + label:hover {
 }
 
 .inputFields {
+  background-color: var(--white-color);
+  border: 1px solid var(--input-shadow);
+  color: var(--white-color);
   box-shadow: 0 1px 1px var(--brand-primary);
+  width: 375px;
+  padding-right: 40px; /* Make room for the button */
+  border-radius: 8px; /* Match the image's rounded corners */
+}
+
+.inputFields:focus {
+  background-color: var(--white-color) !important;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`